blog/20111224new-wiki-and-new-distributions-supported.md

```diff
@@ -19,19 +19,19 @@ We'll announce it here when it's ready.
 
 We also have future plans for a Mate forum, but that is at a very early stage.
 
-For those of you using [Debian](http://www.debian.org/), [Ubuntu](http://www.ubuntu.com),
-and [Linux Mint](http://www.linuxmint.com), you can access the 1.1.x development branch of 
-our packages using the tridex repository. Check out [the download page](http://wiki.mate-desktop.org/download)
-on the wiki for more info. [Linux Mint](http://www.linuxmint.com) also has our 
+For those of you using [Debian](https://www.debian.org/), [Ubuntu](https://www.ubuntu.com),
+and [Linux Mint](https://www.linuxmint.com), you can access the 1.1.x development branch of 
+our packages using the tridex repository. Check out [the download page](https://wiki.mate-desktop.org/download)
```

Comment:
http://wiki.mate-desktop.org/download redirects to https://wiki.mate-desktop.org/download and it doesn't work, dead link